lib/tojson.js

```diff
@@ -73,8 +74,17 @@ exports.toMediaJSON = function (media, session, opts) {
     if (mline.media == 'application') {
         // FIXME: the description is most likely to be independent
         // of the SDP and should be processed by other parts of the library
+
+        var sctpLine = parsers.findLine('a=sctp-port', lines);
+        var sctpPort = sctpLine && sctpLine.split('sctp-port:')[1];
+        var maxMessageSizeLine = parsers.findLine('a=max-message-size', lines);
+        var maxMessageSize = maxMessageSizeLine && maxMessageSizeLine.split('max-message-size:')[1];
+
         content.application = {
-            applicationType: 'datachannel'
+            applicationType: 'datachannel',
+            port: mline.port,
+            sctpPort: sctpPort,
+            maxMessageSize: maxMessageSize
```

Comment:
this is transported via xep-0343 which would need to be updated along with the stanza definitions?